settings.py

```diff
@@ -151,7 +151,8 @@
 SESSION_ENGINE = "django.contrib.sessions.backends.cached_db"
 
 # time in minutes before forced logout due to inactivity
-INACTIVITY_TIMEOUT = 30
+INACTIVITY_TIMEOUT = 60 * 24 * 14
```

Comment:
Yeah. Didn't mean it as a necessary thing. I think this is fine for now.

I usually prefer not adding things to settings that aren't meant to be overridden or used as a configuration.